fcrepo-http-api/src/main/java/org/fcrepo/http/api/FedoraLdp.java

```diff
@@ -389,7 +389,16 @@ public Response createOrReplaceObjectRdf(
         }
 
         try {
-            session.save();
+            // prevent SN-Sibling nodes from creating.
+            synchronized (LOGGER) {
```

Comment:
Yes, they could. In no way does that prescribe locking on a static field in a JAX-RS resource that is not singleton-scoped. There is no possible diagnosis that leads to "lock on a static field" from that beginning. What is more, the fact that clients can misbehave in the way you describe, as they certainly can, is 1) not ameliorated by this locking and 2) absolutely _not_ something for which this code needs to account to begin with.

Can you actually demonstrate (with a test case or at least a careful bug report) the problem you are imagining here and demonstrate (with a test case) that your mechanism solves it?